web/src/__tests__/ProfileCreate.test.tsx

```diff
@@ -14,14 +14,14 @@
 // limitations under the License.
 //
 
-import { act } from "react-dom/test-utils";
+// import { act } from 'react-dom/test-utils';
 // import { create } from "react-test-renderer";
-import React from 'react';
-import { Provider } from 'react-redux';
-import { store } from '../redux/store';
-import ProfileCreate from '../views/ProfileCreate';
+// import React from 'react';
+// import { Provider } from 'react-redux';
 import { MsalReactTester } from 'msal-react-tester';
-import { MsalProvider } from '@azure/msal-react';
+// import { MsalProvider } from '@azure/msal-react';
+// import { store } from '../redux/store';
+// import ProfileCreate from '../views/ProfileCreate';
```

Comment:
Please try not to merge in comment out code, and also it seems like you disable this test, you can probably disable/skip this test instead. 
Last, please squash before merge to master branch